pkgs/os-specific/linux/firejail/fix-opengl-support.patch

```diff
@@ -1,7 +0,0 @@
---- a/etc/inc/whitelist-run-common.inc.org	2022-05-07 11:27:32.264849186 +0200
-+++ b/etc/inc/whitelist-run-common.inc	2022-05-07 11:27:55.577778211 +0200
-@@ -13,3 +13,4 @@
- whitelist /run/systemd/resolve/resolv.conf
- whitelist /run/systemd/resolve/stub-resolv.conf
- whitelist /run/udev/data
-+whitelist /run/opengl-driver	# NixOS
```

Comment:
This was added through https://github.com/netblue30/firejail/commit/1c17d794d1c99d1f76ca4140a989b65167edc890
which is NOT contained in the version nixpkgs currently uses
https://github.com/NixOS/nixpkgs/blob/0db6f20cfeb470c1f778442217c39d8202d591bd/pkgs/os-specific/linux/firejail/default.nix#L14

```
$ git branch --contains 1c17d794d1c99d1f76ca4140a989b65167edc890
master
```